### PR TITLE
(fix) Fix lab order basket on tablet

### PR DESCRIPTION
**Pull request by jwnasambu**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Changed the background colour of the lab Order Basket to white to match this design https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/62c6bb93af241e1aa691d4fc

## Screenshots
**Before making changes**
Screenshot 2023-09-19 at 7 01 35 PM

 **After Making Changes**
Screenshot 2023-09-19 at 5 25 24 PM

## Related Issue
https://issues.openmrs.org/browse/O3-2324

### CODE REVIEW

**Comment by jwnasambu** [comment]
Just learned the Lab order is on another repository that is why am closing this PR

File: packages/esm-patient-labs-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-panel.scss [hunk 1]
```diff
@@ -16,21 +16,14 @@
   margin-bottom: spacing.$spacing-03;
 }
 
-.desktopHeading {
+.desktopHeading, .tabletHeading {
```
Comment (ibacher) [inline packages/esm-patient-labs-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-panel.scss:19]: I think if this requires re-styling the background, we should just collapse this class into `.heading` and apply it unconditionally.

File: packages/esm-patient-labs-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-panel.scss [hunk 2]
```diff
@@ -55,6 +48,12 @@
   border-left: 4px solid #6929c4;
 }
 
+.tabletTile {
```
Comment (ibacher) [inline packages/esm-patient-labs-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-panel.scss:51]: Shouldn't `.tabletTile` be removed from the previous class if we're just adding a subset of properties here. Alternatively, if we need the styles above, we should collapse `.desktopTile` and `.tabletTile` into `.tile` applied unconditionally.

**Review by ibacher — CHANGES_REQUESTED** [review]
Thanks for this @jwnasambu! A couple of simplifications and I think this should be ready-to-go.

**Comment by jwnasambu** [comment]
@ibacher thanks so much for the clear explanation.  Am so glad something where not making sense to me but through the review comments I have got the insight. Basing on this PR https://github.com/openmrs/openmrs-esm-patient-chart/pull/1359/files, it like @brandones covered all that. If its okay with can close this PR. Am closing other commits too which were related to the issues.

**Pull request closed without merging**